chat.js

```diff
@@ -12,35 +13,95 @@ const chatLog = [];
 let first_person = '';
 let second_person = '';
 let current_person = '';
+let fileChoosen = '';
 
-rl.question('Enter first person name: ', (name1) => {
-    first_person = name1;
-    rl.question('Enter second person name: ', (name2) => {
-        second_person = name2;
-        current_person = first_person;
-        chat();
-    });
+rl.question('Continue from previous chat? y/n :', (cont) => {
+	if (cont === "y") {
```

Comment:
**suggestion:** we need to make that case-insensitive, as my first habit personally is to answer with <kbd>Shift</kbd>+<kbd>Y</kbd> 😅.

So either

```suggestion
	if (cont.toLowerCase() === "y") {
```

Or
```suggestion
	if (cont === "y" || cont === "Y") {
```

should work

Reply:
I switched it to the second suggested change. Works well now !!